manual-tests.md

```diff
@@ -231,3 +231,19 @@
 
     * Check that it should end ok
     * Check in the Web that the description is now what the YAML says
+
+
+# Test push binary metadata with conflicts
+
+1. 'snapcraft build' a simple snap
```

Comment:
this should be `snapcraft snap`, `build` is an intermediate command

Reply:
Right, fixing here and the previous one.